forge-gui/res/cardsfolder/upcoming/geths_summons.txt

```diff
@@ -0,0 +1,9 @@
+Name:Geth's Summons
+ManaCost:2 B B
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TgtPrompt$ Choose target creature card in your graveyard | SubAbility$ DBReturnCorrupted | ValidTgts$ Creature.YouCtrl | SpellDescription$ Return up to one target creature card from your graveyard to the battlefield.
+SVar:DBReturnCorrupted:DB$ ChangeZone | Origin$ Graveyard | DefinedPlayer$ Opponent.IsCorrupted | TgtZone$ Graveyard | GainControl$ True | Destination$ Battlefield | TgtPrompt$ Choose target card in an opponent's graveyard | ChangeNum$ 1 | ValidTgts$ Creature.OppCtrl | SpellDescription$ Corrupted — For each opponent who has three or more poison counters as you cast this spell, put up to one target creature card from that player’s graveyard onto the battlefield under your control.
+SVar:OneEach:PlayerCountOpponents$Amount
+DeckNeeds:Ability$Graveyard
+DeckHints:Ability$Graveyard & Keyword$Toxic|Poisonous|Infect
+Oracle:Return up to one target creature card from your graveyard to the battlefield.\nCorrupted — For each opponent who has three or more poison counters as you cast this spell, put up to one target creature card from that player’s graveyard onto the battlefield under your control.
```

Comment:
the problem is "as you cast" looks in the past, so your solution isn't good enough unfortunately

Reply:
Oh yeah didn't realize, I guess this needs big changes ? I will remove this one